src/componentsUniversal/header/header.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../stylesUniversal/mixins/mixins.scss';
+
+.header {
+  padding-top: 62px;
+  margin-bottom: 54px;
+  &__top {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  &__button {
+    width: 150px;
+    height: 40px;
+  }
+}
+
+.menu {
+  &__list {
+    display: flex;
+    list-style-type: none;
+    align-items: baseline;
+  }
+  &__item {
+    display: flex;
+    margin-right: 24px;
+  }
+  &__link {
+    &-first {
+      width: 86px;
```

Comment:
it's better to have 1 common class for links, without numbering and give them common width and align it's content whithin the component